dist/bin-native-overrides/cli-common-platform

```diff
@@ -1,16 +1,3 @@
 #!/usr/bin/env bash
 
-if [[ ${cygwin-} || ${mingw-} || ${msys-} ]]; then
-    SCALA_CLI_VERSION=""
-    # iterate through lines in VERSION_SRC
-    while IFS= read -r line; do
-    # if line starts with "version:=" then extract the version
-    if [[ "$line" == cli_version:=* ]]; then
-        SCALA_CLI_VERSION="${line#cli_version:=}"
-        break
-    fi
-    done < "$PROG_HOME/EXTRA_PROPERTIES"
-    SCALA_CLI_CMD_BASH=("\"$PROG_HOME/bin/scala-cli\"" "--cli-version \"$SCALA_CLI_VERSION\"")
-else
-    SCALA_CLI_CMD_BASH=("\"$PROG_HOME/bin/scala-cli\"")
-fi
+SCALA_CLI_CMD_BASH=("\"$PROG_HOME/bin/scala-cli\"")
```

Comment:
I wonder do we really need this `cli-common-platform` files anymore. There seems to be exactly 1 usage for each of them

Reply:
We need to keep these 2 files separated because we have different content depending on what we are building. If we build universal archives (with jars and no native binaries), we will be using https://github.com/scala/scala3/blob/d48ba033d8128e04727dbee88a5a8eb3fa98d26d/dist/bin/cli-common-platform and https://github.com/scala/scala3/blob/main/dist/bin/cli-common-platform.bat. While if we are building archives with native binaries, we will be using https://github.com/scala/scala3/d48ba033d8128e04727dbee88a5a8eb3fa98d26d/main/dist/bin-native-overrides/cli-common-platform and https://github.com/scala/scala3/blob/d48ba033d8128e04727dbee88a5a8eb3fa98d26d/dist/bin-native-overrides/cli-common-platform.bat